metric/s006.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+"""
+populates referral test urgency from genomic record
+for all patients
+"""
+
+from models import gr
+
+CONFIG = {
+    'concept_code': 'test_urgency',
+    'destination_column': 'value_cid:priority'
+}
+
+def get_data(s):
+    """
+    source data and return list of dictionaries
+    """
+
+    q = s.query(gr.referral_participant.patient_uid,
+                gr.concept.concept_code.label('metric')
+    ).\
+    
+    join(gr.referral,
+             gr.referral_participant.referral_uid ==
+             gr.referral.uid)
+    
+    join(gr.concept,
+             gr.referral.priority_cid ==
+             gr.concept.uid)
```

Comment:
metrics can be at a variety of levels, it doesn't always have to be at patient level. Test urgency is at the referral level so it makes sense to have the metric at referral level too rather than use referral participant.patient_uid which would introduce a lot of duplicate rows (since patients can be in multiple referrals), therefore...
```suggestion
    q = s.query(gr.Referral.uid,
                gr.Concept.concept_code.label('metric')
    ).\
    join(gr.concept,
             gr.referral.priority_cid ==
             gr.concept.uid)
```